src/schema/sources.ts

```diff
@@ -361,6 +362,11 @@ export const typeDefs = /* GraphQL */ `
       """
       featured: Boolean
 
+      """
```

Comment:
It shouldn't be user facing

Reply:
Right. Just used it for testing. We can remove it.

Reply:
This one is for filtering the squad directory when asking for the sources.